packages/tools/devtools/devtools-browser-extension/src/content/ContentScript.ts

```diff
@@ -41,6 +41,23 @@ if (window === undefined) {
 
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 
+const contentScriptPort = browser.runtime.connect({ name: "content-background-channel" });
```

Comment:
I think we need to find a way to share this connection between our new logic here and the existing logic below - otherwise this script is managing two independent handles to the background script. 

Previously, we were waiting for the background script to initiate the connection with this script (when requested by the devtools script), but now we want to initiate the connection immediately when this script runs. That should be fine, but we probably want to refactor the existing logic to do its setup immediately too, rather than waiting for a request from the background (`browser.runtime.onConnect`).